config/checkstyle-input-suppressions.xml

```diff
@@ -2160,42 +2160,6 @@
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterSynchronized.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllTokens.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllTokens.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]whitespace[\\/]nowhitespaceafter[\\/]InputNoWhitespaceAfterTestAllowLineBreaks.java"/>
```

Comment:
@mdxabu  ,
It is recommended to try picking a specific module and working on cleaning it up instead. 
I don't mean this is wrong, but it is surely confusing to review :)

Reply:
Ok, I'll work on correct way on upcoming contributions